src/main/webapp/about.jsp

```diff
@@ -58,20 +59,40 @@
       <p> <br>
         <strong>Meet the Team!</strong>
         <ul>
+            <table cellpadding = "10" cellspacing = "1">
+              <tr><td align="center"><img src="About-IMG/Default-Profile-IMG.png" width = "150" height = "150"><br>
+                <a href="https://github.com/MarcelRG" target="_blank"><i style="color: #000000" class="fa fa-github" aria-hidden="true"></i></a>
 
-        		 <img src="Default-Profile-IMG.png" width = "150" height = "150" style = "float:left">
-        		<li> <strong>Marcel Gonzalez</strong>
-        		Currently studying Computer Science at Tecnologico de Monterrey </li> <br> <br> <br>
+                <a href="ADD LINK HERE!!!!" target="_blank"><i style="color: #000000" class="fa fa-linkedin-square" aria-hidden="true"></i></a>
 
+                <br><br></td>
+                <td><strong>Marcel Gonzalez</strong>
+                Currently studying Computer Science at Tecnologico de Monterrey<br><br><br></td></tr>
+              
 
-        		<br> <img src="Default-Profile-IMG.png" width = "150" height = "150" style = "float:left">
-        		<li> <strong>Ayline Villegas</strong>
-        		Currently studying Computer Science at The University of Illinois at Urbana Champaign </li> <br> <br> <br>
+              <tr><td align="center" ><img src="About-IMG/Ayline-Profile.gif" width = "150" height = "150" style = "border-radius: 50%"><br>
+                <a href="https://www.facebook.com/ayline.villegas" target="_blank"><i style="color: #000000" class="fa fa-facebook-square" aria-hidden="true"></i></a>
 
+                <a href="https://www.instagram.com/ayline_villegas/" target="_blank"><i style="color: #000000" class="fa fa-instagram" aria-hidden="true"></i></a>
 
-        		<br> <img src="Anthony-Profile-IMG.jpg" width = "150" height = "150" style = "float:left;border-radius:50%"">
-        		<li> <strong>Anthony Beltran</strong>
-        		Currently studying Computer Science at The University of Illinois at Chicago. Passions include: nature, AI, and food! <a href = "https://www.linkedin.com/in/abeltran1804/">linkedin</a> </li> <br> <br> <br>
+                <a href="https://github.com/aylinev2" target="_blank"><i style="color: #000000" class="fa fa-github" aria-hidden="true"></i></a>
+
+                <a href="https://www.linkedin.com/in/aylinev/" target="_blank"><i style="color: #000000" class="fa fa-linkedin-square" aria-hidden="true"></i></a>
+
+                <br><br></td>
+                <td><a href="http://aylinev2.github.io"><strong>Ayline Villegas</strong></a><br>
+                Currently studying Computer Science at The University of Illinois at Urbana Champaign. Passions include: cats, baking, and music!<br><br><br></td></tr>
+
+
+              <tr><td align="center"><img src="About-IMG/Anthony-Profile-IMG.jpg" width = "150" height = "150" style = "border-radius:50%""><br>
```

Comment:
If you are up for it, you could also put the width, height, radius, and float into CSS so it stays the same for all images.

Reply:
thanks for the tips! I just made these fixes so ill go ahead and let you review them before I merge